.travis.yml

```diff
@@ -82,8 +82,8 @@ before_script:
     fi
 
 script:
-  - coverage run -m pytest  # Run the tests and check for test coverage.
+  - cd bluesky_kafka
```

Comment:
This shouldn't be needed. If it is, let's fix it with `pytest.ini`. See for example https://github.com/bluesky/resource-health-check/blob/master/pytest.ini. But naively I would expect this to work without `cd` or `pytest.ini`.

Reply:
I don't usually have to play tricks like this with pytest but something was going on that I do not understand. I agree `pytest.ini` is much better than what I have done.

Reply:
Bump on this.